lite/examples/sound_classification/android_compose/README.md

```diff
@@ -0,0 +1,44 @@
+# Sound Classifier Android sample.
+
+This Android application demonstrates how to classify sound on-device. It uses:
+
+* [TFLite Task Library](https://www.tensorflow.org/lite/inference_with_metadata/task_library/overview)
+* [YAMNet](https://tfhub.dev/google/lite-model/yamnet/classification/tflite/1),
+an audio event classification model.
+
+## Requirements
+
+*   Android Studio 2020.3.1 Patch 2 (installed on a Linux, Mac or Windows machine)
```

Comment:
Could you rename it to Android Studio Arctic Fox. If the patch 2 is needed, just append to it

Reply:
Oh, thanks for the good point.